src/raster/r.mblend/README.md

```diff
@@ -3,7 +3,7 @@ r.mblend
 
 Draft implementation of the [Mblend algorithm](http://www.sciencedirect.com/science/article/pii/S0098300416300012) as an add-on for [GRASS 7](https://grass.osgeo.org/grass7/). 
 
-Manual and detailed description is available at the [GRASS manual pages](https://grass.osgeo.org/grass74/manuals/addons/r.mblend.html).
+Manual and detailed description is available at the [GRASS manual pages](https://grass.osgeo.org/grass7/manuals/addons/r.mblend.html).
```

Comment:
```suggestion
Manual and detailed description is available at the [GRASS manual pages](https://grass.osgeo.org/grass8/manuals/addons/r.mblend.html).
```

Reply:
Addressed in 59513d57aadcdc43a97291fcadd06ffc9b55fd62